server/game/baseability.js

```diff
@@ -136,7 +136,8 @@ class BaseAbility {
      */
     canPayCosts(context) {
         let cost = this.cost.concat(context.player.getAdditionalCosts(context));
-        return cost.every((cost) => cost.canPay(context));
+        let aggregatedCost = {};
```

Comment:
This is a bit confusing. Can there be anything other than amber on this? If not just make it be amber. If so, at least initialise amber to 0 in here

Reply:
In theory, canPay is agnostic. Technically, it could be amber, chain, key (see costs.js)